README.md

```diff
@@ -113,6 +125,17 @@ Pull requests with the label "feature" or "fix" will now be grouped together:
 
 <img src="design/screenshot-2.png" alt="Screenshot of generated draft release with categories" width="586" />
 
+## Non Semver Versions
```

Comment:
@TimonVS given we already have first-class concepts of major, minor and patch… reading this in the readme now feels very abstract. Unless we have some other use case examples?

Reply:
@toolmantim I agree that it might feel a little abstract. I think it's quite a niche feature so maybe we can push it more to the background as a power user feature? The benefit of this more abstract approach is that we can add support for [CalVer](http://calver.org/) easily. I don't know about any other widely used versioning systems though.

Reply:
Into the background should I move it down inside the README?

Reply:
Yes would be super easy to add support for CalVer :sweat_smile: